pandas/io/parquet.py

```diff
@@ -184,6 +204,12 @@ def write(
             )
 
     def read(self, path, columns=None, **kwargs):
+        use_nullable_dtypes = kwargs.pop("use_nullable_dtypes", False)
```

Comment:
we should have a global option to turn this on (pls add an issue for this)

Reply:
I think this generally worth it, if you can add an issue for this / PR welcome too! (bot blocking for this PR)